backend/chunker/chunker.go

```diff
@@ -1123,11 +1123,30 @@ func (f *Fs) newChunkingReader(src fs.ObjectInfo) *chunkingReader {
 	return c
 }
 
+// isLocalObject() returns true if source wraps a local file.
+// Local files provide any hashsum but hashing may take long and delay transfer.
+// Remote sources keep supported hashsum in metadata and give it instantly.
+func isLocalObject(src fs.ObjectInfo) bool {
+	if srcObj, ok := src.(fs.Object); ok {
```

Comment:
You could also use the very recently added Features().SlowHash

This is advertised by the `sftp` and `local` backends only.

What do you think?

Reply:
Agreed. The **SlowHash** name more correctly expresses the intention.

Reply:
I don't see Local and Sftp backends advertising SlowHash=true on the tip of master.
Is it committed yet?

Reply:
We can consider another approach: `func (fs.ObjectInfo) Hash2(hashType, onlyFast bool)` or extend existing `fs.ObjectInfo.Hash` with second parameter.
Also, even in view of `Features().SlowHash` we still need a local function `isSlowHash` since we need to unwrap `ObjectInfo` in case of wrapped remotes: `fs` provides unwrapper for objects but not for object info.

Reply:
SlowHash is in master now.

>  fs provides unwrapper for objects but not for object info.

I had this in another branch so I pulled it into master ae888dbacf64a726a4c354981a38198314e5929c

Reply:
I actually merged the change now (forgot to push it before) 51a230d7fd0f8bcca5d205554e708b81e5162ca7

Maybe that should be changed to assert an ObjectUnwrapper like your version?